pdr_backend/lake/gql_data_factory.py

```diff
@@ -50,37 +55,32 @@ def __init__(self, ppss: PPSS):
 
         # configure all tables that will be recorded onto lake
         self.record_config = {
-            "pdr_predictions": {
-                "fetch_fn": get_pdr_predictions_df,
-                "schema": predictions_schema,
-                "config": {
-                    "contract_list": contract_list,
-                },
-            },
-            "pdr_subscriptions": {
-                "fetch_fn": get_pdr_subscriptions_df,
-                "schema": subscriptions_schema,
-                "config": {
-                    "contract_list": contract_list,
-                },
-            },
-            "pdr_truevals": {
-                "fetch_fn": get_pdr_truevals_df,
-                "schema": truevals_schema,
-                "config": {
-                    "contract_list": contract_list,
-                },
+            "tables": {
```

Comment:
Please create a table registry outside of GQLDataFactory so we can define all tables in there.
Any other class should be able to access the registry and retrieve a table.

Reply:
I don't know where to move them within this PR. What I'm thinking is moving them to the ETL object which is responsible to create and update all the tables, but that is created inside another PR.

Reply:
Let's handle this inside the ETL